recipes/snappy/bld.bat

```diff
@@ -0,0 +1,12 @@
+cd %SRC_DIR%
+git clone https://github.com/kmanley/snappy-msvc
```

Comment:
I guess this is not part of the source code.
Can we clone a specific hash? (Although it seems that nothing is updated in almost 4 years!)

PS: Maybe we should just download those files and ship with the recipe. Just adding a comment about that repo.